# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../zaptel/split_make_install.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Note: Just build and install userspace stuff of zaptel 
2006-04-02 Christian Wiese
2006-07-01 re-diffed for 1.2.6 by Aldas Nabazas (Baldzius)

diff -Nur zaptel-1.2.6-orig/Makefile zaptel-1.2.6/Makefile
--- zaptel-1.2.6-orig/Makefile	2006-05-22 23:11:36.000000000 +0000
+++ zaptel-1.2.6/Makefile	2006-07-01 16:55:11.000000000 +0000
@@ -128,6 +128,8 @@
 
 all: $(BUILDVER) $(LIBTONEZONE_SO)
 
+bins: $(BINS)
+
 linux24: prereq $(MODULESO) $(BINS)
 
 linux26: prereq $(BINS)
@@ -300,14 +302,7 @@
   endif
 endif
 
-install: all devices
-	install -D -m 755 ztcfg $(INSTALL_PREFIX)/sbin/ztcfg
-	if [ -f sethdlc-new ]; then \
-		install -D -m 755 sethdlc-new $(INSTALL_PREFIX)/sbin/sethdlc; \
-	elif [ -f sethdlc ]; then \
-		install -D -m 755 sethdlc $(INSTALL_PREFIX)/sbin/sethdlc ; \
-	fi
-	if [ -f zttool ]; then install -D -m 755 zttool $(INSTALL_PREFIX)/sbin/zttool; fi
+install-modules: 
 ifeq ($(BUILDVER),linux26)
 	for x in $(MODULESKO); do \
 		rm -f $(INSTALL_PREFIX)/lib/modules/$(KVERS)/extra/$$x ; \
@@ -322,6 +317,14 @@
 		rm -f $(INSTALL_PREFIX)/lib/modules/$(KVERS)/misc/wcfxsusb.o; \
 	fi; \
 	rm -f $(INSTALL_PREFIX)/lib/modules/$(KVERS)/misc/wcfxs.o
+install: 
+	install -D -m 755 ztcfg $(INSTALL_PREFIX)/sbin/ztcfg
+	if [ -f sethdlc-new ]; then \
+		install -D -m 755 sethdlc-new $(INSTALL_PREFIX)/sbin/sethdlc; \
+	elif [ -f sethdlc ]; then \
+		install -D -m 755 sethdlc $(INSTALL_PREFIX)/sbin/sethdlc ; \
+	fi
+	if [ -f zttool ]; then install -D -m 755 zttool $(INSTALL_PREFIX)/sbin/zttool; fi	
 	install -D -m 755 $(LIBTONEZONE_SO) $(INSTALL_PREFIX)/usr/lib/$(LIBTONEZONE_SO).$(LIBTONEZONE_SO_MAJOR_VER).$(LIBTONEZONE_SO_MINOR_VER)
 	[ `id -u` = 0 ] && /sbin/ldconfig || :
 	rm -f $(INSTALL_PREFIX)/usr/lib/$(LIBTONEZONE_SO)
